apps/api/src/server/applications/examination-timetable-items/examination-timetable-items.routes.js

```diff
@@ -62,12 +62,12 @@ router.post(
             required: true
         }
         #swagger.responses[200] = {
-            description: 'List of examination timetable items',
+            description: 'Examination timetable item',
             schema: { $ref: '#/definitions/ExaminationTimetableItemsResponseBody' }
```

Comment:
the response schema definition is now singular, also this endpoint returns an array of ExaminationTimetableItem.
description of the endpoint should say that it creates a new exam timetable item record, not a get.
Also the body parameters description is incorrect - it is passed a set of exam timetable item properties